css/order.css

```diff
@@ -398,3 +364,191 @@ svg {
   text-align: center;
   transition: 300ms all ease-in-out;
 }
+
+#bnomics-address-input,
+#bnomics-amount-input {
+  padding: 5px;
+  max-width: 190%;
+  color: currentColor;
+  margin: 0;
+}
+
+.bnomics-order-address {
+  align-items: center;
+  max-width: 580px;
+  margin-left: 8px;
+  margin-right: 8px;
+  text-align: left;
+}
+
+.bnomics-or-divider {
+  position: relative;
+  text-align: center;
+  vertical-align: middle;
+  font-size: var(--global--font-size-xs);
+  font-weight: 700;
+  opacity: 0.5;
+
+}
+
+.bnomics-or-divider::before,
+.bnomics-or-divider::after {
+  content: '';
+  display: block;
+  width: 1px;
+  background: currentColor;
+  height: 36%;
+  position: absolute;
+  left: 50%;
+  transform: translateX(-50%);
+}
+
+.bnomics-or-divider::before {
+  top: 16px;
+}
+
+.bnomics-or-divider::after {
+  bottom: 16px;
+}
+
+
+
+.bnomics-order-panel {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  
+  padding-left: 1rem;
+  padding-right: 1rem;
+}
+
+.bnomics-header-container,
+.bnomics-footer-container {
+  margin-left: auto;
+  margin-right: auto;
+  width: 100%;
+  max-width: 480px;
+  margin-top: 0.5em;
+  margin-bottom: 0.5em;
+  border-collapse: separate;
+  font-size: 20px;
+  border-collapse: separate;
+  border-spacing: 10px 0px;
+  padding: 5px 10px;
+}
+
+.bnomics-footer {
+  display: flex;
+  align-items: center;
+  position: relative;
+  justify-content: space-between;
+  padding-bottom: 2px;
+  max-width: 160%;
+}
+
+.blockonomics_checkout_table {
+  margin: 0;
+}
+
+
+
+#blockonomics_checkout .blockonomics_checkout_table {
+  border-spacing: initial;
+}
+
+.bnomics-checkout-row td {
+  padding: 0;
+ vertical-align: middle; 
+}
+
+.bnomics-order-address label,
+.bnomics-qr a {
+  font-size: var(--global--font-size-xs);
+}
+
+.bnomics-footer-divider {
+  display: none;
+}
+
+table.blockonomics_checkout_table tbody td {
+  background-color: transparent !important;
+}
+
+
+
+@media screen and (max-width: 600px) {
```

Comment:
At 600px, there is no border for both the outer table td's and the inner table td's.